css/home.css

```diff
@@ -1,4 +1,52 @@
 /* Home page specific styles */
 
```

Comment:
This is not part of your submission but since there is now code in the file can you remove this empty lines?

Reply:
Will do